docs/creating-manifest/placeholders.md

```diff
@@ -593,6 +601,51 @@ onInstall:
 The results on the screen below:
 ![comparison](/img/comparison.png)
 
+## Engine Placeholder
+The *${engine}* placeholder returns a Cloud Scripting engine version that is supported by the platform the manifest is executed on.  
+
+`${engine}` - CS engine version  
+
+It is can be complemented with function placeholders *[${fn.compareEngine(version)}](https://docs.cloudscripting.com/creating-manifest/placeholders/#function-placeholders)* and *[${fn.compare(version1, version2)}](https://docs.cloudscripting.com/creating-manifest/placeholders/#function-placeholders)* that can be used to determine whether JPS manifest is supported by the platform’s engine version or not.  
+
+@@@
+```yaml
+type: install
+name: Ability co compare CS engine versions
+
+onInstall:
+- assert:
+  - "'${engine}'.split('.').length > 0"
+  - "'${fn.compare}' == 0"
+  - "'${fn.compare(5.4.1, 5.4.2)}' == -1"
+  - "'${fn.compare(5.5, 5.4.2)}' == 1"
+  - "'${fn.compare(5.4.0, 5.4.0.0)}' == 0"
+  - "'${fn.compareEngine}' == 0"
+  - "'${fn.compareEngine(1.5.1)}' == 1"
+  - "'${fn.compareEngine(1000000)}' == -1"
+```
+```json
+{
+  "type": "install",
+  "name": "Ability co compare CS engine versions",
+  "onInstall": [
+    {
+      "assert": [
+        "'${engine}'.split('.').length > 0",
+        "'${fn.compare}' == 0",
+        "'${fn.compare(5.4.1, 5.4.2)}' == -1",
+        "'${fn.compare(5.5, 5.4.2)}' == 1",
+        "'${fn.compare(5.4.0, 5.4.0.0)}' == 0",
+        "'${fn.compareEngine}' == 0",
+        "'${fn.compareEngine(1.5.1)}' == 1",
+        "'${fn.compareEngine(1000000)}' == -1"
+      ]
+    }
+  ]
+}
+```
+@@!
+
```

Comment:
describe also the following example: https://jelastic.team/browse/JE-43816?focusedCommentId=229848&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-229848

Reply:
seems example contains an error and the if condition result should be compared with '1' instead of '-1'. I changed  it in the commit.

Reply:
done